roseau/load_flow/models/loads/loads.py

```diff
@@ -16,562 +15,222 @@
 logger = logging.getLogger(__name__)
 
 
-class AbstractLoad(Element, JsonMixin, metaclass=ABCMeta):
-    """An abstract class to depict a load."""
-
-    _power_load_class: Optional[type["PowerLoad"]] = None
-    _delta_power_load_class: Optional[type["DeltaPowerLoad"]] = None
-    _impedance_load_class: Optional[type["ImpedanceLoad"]] = None
-    _delta_impedance_load_class: Optional[type["DeltaImpedanceLoad"]] = None
-    _admittance_load_class: Optional[type["AdmittanceLoad"]] = None
-    _delta_admittance_load_class: Optional[type["DeltaAdmittanceLoad"]] = None
-    _flexible_load_class: Optional[type["FlexibleLoad"]] = None
-
-    def __init__(self, id: Any, n: int, bus: Bus, **kwargs) -> None:
-        """Load constructor.
+def _temporarily_check_only_s_or_z(
+    s: Optional[Sequence[complex]], i: Optional[Sequence[complex]], z: Optional[Sequence[complex]]
+) -> None:
+    # Temporarily block loads that are not yet available until we implement the full load equation
+    # Remove this check later
+
+    # Constant current loads are not implemented
+    if i is not None:
+        raise RoseauLoadFlowException(
+            msg=(
+                "Load currently expects s (constant power) or z (constant impedance), "
+                "i (constant current) is not implemented"
+            ),
+            code=RoseauLoadFlowExceptionCode.BAD_LOAD_TYPE,
+        )
+
+    # Mixed constant power and constant impedance loads are not implemented
+    if s is not None and z is not None:
+        raise RoseauLoadFlowException(
+            msg="Load currently expects only s (constant power) or z (constant impedance), not both.",
+            code=RoseauLoadFlowExceptionCode.BAD_LOAD_TYPE,
+        )
+    elif s is None and z is None:
+        raise RoseauLoadFlowException(
+            msg="Load currently expects either s (constant power) or z (constant impedance), non given.",
+            code=RoseauLoadFlowExceptionCode.BAD_LOAD_TYPE,
+        )
```

Comment:
I don't understand what are the mixed loads we are talking about, is this something like 30% s, 30% z, 40% i ?

Reply:
Le plus simple serait de laisser trois types de charge: une à puissance constante, une à intensité constante et une troisième à impédance fixe. L'utilisateur voulant mélanger ces charges n'a qu'à connecter trois charges à un même bus. Cela présente quelques avantages:
1. C'est simple!
2. Si on mélange les charges, on va forcément avoir des demandes du type: "la partie ma charge à puissance constante tire combien de courant?". Réponse : "avec la charge combinée, on n'a que la somme qui est disponible... "

La partie à simplifier de la charge est sur la connexion : je souhaite fusionner les charge en delta et en étoile pour n'avoir qu'une classe par type de charge (puissance, intensité et impédance)